content/md/retailer-onboarder/first-steps/synchronization.md

```diff
@@ -33,8 +33,8 @@ Catalog structure updates are also synchronized with the relevant Supplier Onboa
 As soon as you link a product to a supplier and you set a supplier reference for a product, the attributes mentioned above will be synchronized with the relevant Supplier Onboarder. Therefore, the product will be displayed in the relevant supplier Onboarder product grid.
 
 # Assets
-When you assign a product to a supplier, all the assets linked to this product will be sent. The supplier will be able to pick within the asset of the products or to upload new assets for a product.
+When you assign a product to a supplier, all the assets linked to this product will be sent. The supplier will be able to pick in the asset collection linked to the products or to upload new media files for a product.
 
 :::warning
-Only media files within assets can be synchronized with the Onboarder.
+Only media files of an assets linked to the asset collection of a product are synchronized with the Onboarder.
```

Comment:
Je ne comprends pas la phrase. Que veux tu dire?

Reply:
Ce que je veux dire c'est qu'on ne va envoyer au supplier **que** les media files dans les assets de la famille d'asset lié à l'asset collection. Tous les autres types d'attribute d'asset ne sont pas envoyés aux suppliers.

Reply:
oh ok! j'avais pas compris :) du coup, je pense que tu peux dire un truc du genre  "we don't synchronize all the attributes of the asset. Only media files (and asset code/filename?) of the assets in the asset collection are synchronized with the Onboarder" ?